sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -276,8 +290,39 @@ export class ApplicationService extends RecordDataModelService<Application> {
       application.currentAssessment.updatedAt = now;
 
       await applicationRepository.save([application, newApplication]);
+      // Create a notification if the application is edited for the 5th time.
+      const applicationsCount = await applicationRepository.count({
+        where: { applicationNumber: newApplication.applicationNumber },
+      });
+      if (
+        applicationsCount ===
```

Comment:
I was expecting something like saving the application number as metadata and identifying if the application ever sent this notification after the count was exceeded.